Heading note added to backwards incompatible changes (#44407)
home-assistant/home-assistant.github.io: source/_posts/2026-04-01-release-20264.markdown

We always make sure to document these changes to make the transition as easy as possible for you. This release has the following backward-incompatible changes:

> {% details "Dashboards"}

Author: ```suggestion
{% details "Dashboards" %}
```

The default grid row size for the heading card changed from 1 to auto, making it shorter (half row). This is only noticeable for headings placed between other cards. Headings at the top of a section are unaffected thanks to the updated grid padding.

> The user can be restored the previous row gap between section by settings the `ha-view-sections-row-gap` theme variable to `8px` if more compact layout is wanted.

Reviewer: I like this suggestion, as it removes "the user". We should always direct at the reader directly, so the suggested text is better.

That said, I think then general reader has no idea what "ha-view-sections-row-gap" is or what to do with this information. I feel like this is missing an intro? Like: if you develop custom themes.. (or something)

> The user can be restored the previous row gap between section by settings the `ha-view-sections-row-gap` theme variable to `8px` if more compact layout is wanted.

Author: ```suggestion
The default grid row size for the heading card changed from 1 to auto, making it shorter (half a row). This is only noticeable for headings placed between other cards. Headings at the top of a section are unaffected thanks to the updated grid padding.

If you’re using a custom theme, you can restore the previous spacing between sections by setting the ha-view-sections-row-gap theme variable to 8px for a more compact layout.
```

> If you’re using a custom theme, you can restore the previous spacing between sections by setting the ha-view-sections-row-gap theme variable to 8px for a more compact layout.

Author: ```suggestion
If you’re using a custom theme, you can restore the previous spacing between sections by setting the `ha-view-sections-row-gap` theme variable to `8px` for a more compact layout.
```
